supportedBackends/aws/src/main/scala/cromwell/backend/impl/aws/AwsBatchRuntimeAttributes.scala

```diff
@@ -154,10 +154,26 @@ object AwsBatchRuntimeAttributes {
 object QueueArnValidation {
   lazy val instance: RuntimeAttributesValidation[String] = ArnValidation(AwsBatchRuntimeAttributes.QueueArnKey)
 
-  def configDefaultWomValue(config: Option[Config]): Option[WomValue] = instance.configDefaultWomValue(config)
+  def configDefaultWomValue(config: Option[Config]): Option[WomValue] =
+    instance.configDefaultWomValue(config).filter(ArnValidation.isValidArn)
 }
-// TODO: provide arn regexp validation here
+
 object ArnValidation {
+  // General ARN format can be found here
+  // https://docs.aws.amazon.com/en_us/general/latest/gr/aws-arns-and-namespaces.html#genref-arns
+  // In current form it will not recognize the following ARN as valid due to its complex structure
+  // arn:aws:autoscaling:us-east-1:123456789012:scalingPolicy:c7a27f55-d35e-4153-b044-8ca9155fc467:autoScalingGroupName/my-test-asg1:policyName/my-scaleout-policy
+  // Support for this format greatly complicates the regexp.
+  // Since autoscaling ARN's are not used by Cromwell by now, it is not yet clear how much support of this ARN is needed.
+  /**
+   * @return a function that returns true if the valueString of passed WomValue is a valid AWS ARN and false otherwise
+   */
+  def isValidArn: WomValue => Boolean =
```

Comment:
Taking in consideration your hello note, I guess, the most simple and reasonable way is to name this method as `isValidQueueArn`.

Reply:
At the moment, this method checks the correctness of any `arn`, not just `queueArn`.